ueransim_lb/ueransim-gnb_init.sh

```
@@ -0,0 +1,22 @@
+sed -i 's|MCC|'$MCC'|g' /UERANSIM/config/${COMPONENT_NAME}.yaml
+sed -i 's|TAC|'$TAC'|g' /UERANSIM/config/${COMPONENT_NAME}.yaml
+sed -i 's|NR_GNB_IP|'$NR_GNB_IP'|g' /UERANSIM/config/${COMPONENT_NAME}.yaml
+sed -i 's|AMF_IP|'$AMF_IP'|g' /UERANSIM/config/${COMPONENT_NAME}.yaml
+
+# Add route to loxilb VIP via loxilb container
+# This ensures NGAP traffic goes through the load balancer
+echo "Adding route to loxilb VIP..."
+ip route add 20.20.20.1/32 via 172.22.0.100 || true
+
+./nr-gnb -c ../config/${COMPONENT_NAME}.yaml &
+exec bash $@
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)